### PR TITLE
JSON config

### PR DESCRIPTION
**Pull request by Dahlgren**

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -153,71 +152,22 @@ int main(int argc, char* argv[]) {
...
-    myThread->join();
-    myThread2->join();
-    myThread3->join();
-    myThread4->join();
-    myThread5->join();
-    myThread6->join();
-    myThread7->join();
-
-    /*
-    auto compiledScript = compiler.compileScript("I:/ACE3/addons/advanced_ballistics/functions/fnc_readWeaponDataFromConfig.sqf");
```
Comment (dedmen) [inline src/main.cpp:192]: Thats my testing code :cry: I'd prefer to have this here so I can easily uncomment it for testing

Comment (Dahlgren) [reply src/main.cpp:192]: I'll restore it 🤣

File: README.md [hunk 2]
```diff
@@ -1 +1,29 @@
...
+
+```json
+{
+  "inputDirs": [
+    "T:/x/",
+    "T:/z/ace/",
+    "T:/z/acex/",
+    "T:/a3/"
+  ],
+  "includePath": "T:/",
```
[the remark this answers is not in the document]
Comment (Dahlgren) [reply README.md:15]: Yeah, they should be, I missed the vector param

File: README.md [hunk 3]
```diff
@@ -1 +1,29 @@
...
+  "excludeList": [
```
[the remark this answers is not in the document]
Comment (Dahlgren) [reply README.md:18]: Baby steps! 🐤

**Comment by Dahlgren** [comment]
Changed name of config file, `config.json` -> `sqfc.json`

**Pull request merged**